src/redux/actions/experimentSettings/backendStatus/loadBackendStatus.js

```diff
@@ -63,11 +63,12 @@ const loadBackendStatus = (experimentId) => async (dispatch) => {
     // having to run the pipelines
     if (process.env.NODE_ENV === 'development'
       && (status.gem2s.status === pipelineStatusValues.NOT_CREATED
-        || status.qc.status === pipelineStatusValues.NOT_CREATED)) {
+        || status.pipeline.status === pipelineStatusValues.NOT_CREATED)) {
```

Comment:
did we change this back from `qc` to `pipeline`?

Reply:
It was never changed initially I think, there is a ticket to move the last parts of qc name to pipeline in the api, but right now this was causing the platform to crash in localhost, so I changed it back to `pipeline`